hydrolib/core/dflowfm/extold/parser.py

```diff
@@ -0,0 +1,77 @@
+from pathlib import Path
+from typing import Dict, List
+
+from hydrolib.core.dflowfm.extold.common_io import ORDERED_FORCING_FIELDS
+
+
+class Parser:
+    """Parser class for parsing the forcing data of the old external forcings file to a dictionary to construct the `ExtOldModel` with."""
+
+    @staticmethod
+    def parse(filepath: Path) -> Dict:
+        """Parses the file at the specified path to the forcing data.
+
+        If a line starts with an asterisk (*) it is considered a comment and will not be parsed.
```

Comment:
Some files do support reading and storing comments and others do not. Wouldn't it be nice if all files behaved the same? Should we create a new issue for this?

Reply:
Good point, will ask Jelmer or Arthur.

Reply:
Did they reply?